src/Model/User.php

```diff
@@ -185,6 +185,29 @@ public static function getByEmail($email, array $fields = [])
 		return DBA::selectFirst('user', $fields, ['email' => $email]);
 	}
 
+	/**
+	 * Fetch the user array of the administrator. The first one if there are several.
+	 *
+	 * @param array $fields
+	 * @return array user
+	 */
+	public static function getFirstAdmin(array $fields = [])
+	{
+		$condition = [];
```

Comment:
`$condition` is never used in this method.

Reply:
Not only this - but the functionality had been weird. That's what happens when you perform "cut&paste" at 5 in the morning :-)